### PR TITLE
change limit 1000 to 1,reduce the amount the data

### PR DESCRIPTION
**Pull request by Maxgis**
The limit default value is 1000. I think the num of record is 1.

### CODE REVIEW

**Comment by miraclesu** [comment]
LGTM

**Comment by Maxgis** [comment]
@miraclesu ok, I add test case torommow

**Comment by astaxie** [comment]
@Maxgis test failed. please make sure you have tested in local

**Comment by Maxgis** [comment]
@astaxie updated and test success.

**Comment by astaxie** [comment]
@Maxgis this update looks trick. @miraclesu could you help to refactor this function. I think One need to limit=1. but looks the test case failed.

**Comment by Maxgis** [comment]
@astaxie I think that is is not need to throw ErrMultiRows.It will update the function and test case

**Comment by astaxie** [comment]
@Maxgis yea. I agree. please help to update the function and test case.

**Comment by Maxgis** [comment]
If not not specified collation, different databases return different.

File: orm/orm_queryset.go [hunk 1]
```diff
@@ -192,13 +192,11 @@ func (o *querySet) All(container interface{}, cols ...string) (int64, error) {
...
 // cols means the columns when querying.
 func (o *querySet) One(container interface{}, cols ...string) error {
+	o.limit = 1
 	num, err := o.orm.alias.DbBaser.ReadBatch(o.orm.db, o, o.mi, o.cond, container, o.orm.alias.TZ, cols)
 	if err != nil {
 		return err
 	}
-	if num > 1 {
-		return ErrMultiRows
-	}
```
Comment (astaxie) [inline orm/orm_queryset.go]: if we set limit to 1, and then the num return more than 1. I think should be return the `ErrMultiRows `

**Comment by Maxgis** [comment]
@astaxie You are right

**Pull request merged**